### PR TITLE
Remove PHI from job name and increase job name length

### PR DESCRIPTION
**Pull request by mocsharp**
### Description
* Remove PHI from generated job name.
* Increase length of job name limit to 63 characters

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] All tests passed locally by running `./src/run-tests-in-docker.sh`.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.
- [x] User guide updated.
- [ ] NGC publishing metadata updated.
- [ ] I have updated the changelog

### CODE REVIEW

File: docs/setup/ae-title-job-processor.md [hunk 1]
```diff
@@ -57,3 +57,18 @@ separate association: The 2nd instance is received at T1+4 (where T1 is the time
...
+```
+{Called AE Title}-{Name of the pipeline}-{UTC Time "yyyyMMddHHmmss"}
+```
+
+e.g.
+Given:
+* AE Title: LIVERAET
+* Pipeline: LIVERAI
+```
+LIVERAET-LIVERAI-20211225101030
```
Comment (MMelQin) [inline docs/setup/ae-title-job-processor.md:73]: Two  concerns:
1. Are their going to be conflict if more than one job is created at the same time down to seconds, for the same AET.
2. This means there is no way to associate a job instance to a input dataset. Is there an internal tracking index DB. When issue happens, a admin user then need to download the payload and inspect, if there is no tracking DB to provide the info.

Comment (mocsharp) [reply docs/setup/ae-title-job-processor.md:73]: 1. With *AE Title Job Processor*, it's impossible to have same job created at the same time due to the minimal timeout limit to 5s. 
2. No DB index tracking the instances.  However, user could check the payload input directory.

Comment (MMelQin) [reply docs/setup/ae-title-job-processor.md:73]: This requires the specific AET's job processor run single thread (or multi but with serialized ID creation with no more than 1 per second). 
Are you using UTC time, which it does not appear to be? When local daylight saving time falls back, there could be a overlap causing duplicate.

Comment (mocsharp) [reply docs/setup/ae-title-job-processor.md:73]: Yes, am using UTC time.

Comment (MMelQin) [reply docs/setup/ae-title-job-processor.md:73]: Indeed, as specified, UTC is used.

**Review by MMelQin — APPROVED** [review]

**Pull request merged**